### DISCUSSION TITLE
Why the page is not working it used to work for me before?

### DISCUSSION THREAD

**Discussion by Hamza-Noah** [community/community#22024] [Other Feature Feedback, Questions, & Ideas]
<p>this is the repository link <a href="https://github.com/Hamza-Noah/Portfolio" rel="noopener nofollow ugc">repository </a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/5/8/58e87b30ff2bce3c341bae66fc8c72b09838a5fa.png" data-download-href="/uploads/short-url/cGw6zSC9ziA8FPeZwKWT7hXDG78.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/82677794/181068311-3dacbde4-4d6c-44d4-9081-62ed8cbfb057.png" alt="image" data-base62-sha1="cGw6zSC9ziA8FPeZwKWT7hXDG78" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/5/8/58e87b30ff2bce3c341bae66fc8c72b09838a5fa_2_690x388.png, https://aws1.discourse-cdn.com/github/optimized/3X/5/8/58e87b30ff2bce3c341bae66fc8c72b09838a5fa_2_1035x582.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/5/8/58e87b30ff2bce3c341bae66fc8c72b09838a5fa_2_1380x776.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/8/58e87b30ff2bce3c341bae66fc8c72b09838a5fa_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1080 60.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment, marked as answer]
<p>did you check the Developer Tools at your browser ?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/b/2b2a1fd51342530790a20bd31af376d1896076c1.png" data-download-href="/uploads/short-url/69QMuRtg0WRLls74Q2FctNcLCMx.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181068313-d5deea21-9ae2-408e-be26-f5e661defa3b.png" alt="image" data-base62-sha1="69QMuRtg0WRLls74Q2FctNcLCMx" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/b/2b2a1fd51342530790a20bd31af376d1896076c1_2_690x373.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/b/2b2a1fd51342530790a20bd31af376d1896076c1_2_1035x559.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/2/b/2b2a1fd51342530790a20bd31af376d1896076c1_2_1380x746.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/b/2b2a1fd51342530790a20bd31af376d1896076c1_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 124 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>I’m seeing those, not very sure what happened to your site but one that I’m seeing is this</p>
<p><img src="https://user-images.githubusercontent.com/47092464/181068314-b4c14d9a-0cbd-4bb0-ac7e-4b6ade98fbd2.png" alt="image" data-base62-sha1="maxz38fe7VCOcAclRcZmWHxSZ8c" width="485" height="135"></p>
<p>it will be 404 since there is no directory named as <code>images</code></p>
